rspace/src/main/scala/coop/rchain/rspace/history/RadixStore.scala

```diff
@@ -0,0 +1,43 @@
+package coop.rchain.rspace.history
+
+import cats.effect.Sync
+import cats.syntax.all._
+import coop.rchain.store.KeyValueStore
+import scodec.bits.ByteVector
+
+import java.nio.ByteBuffer
+import scala.language.higherKinds
+
+class RadixStore[F[_]: Sync](store: KeyValueStore[F]) {
```

Comment:
Just a nitpick.  It seems `RadixStore` can be extended as `KeyValueStore`

Reply:
Thanks, I will study such an opportunity

Reply:
Sorry, but I don't understand why to extend it from `KeyValueStore`?
Class `RadixStore` creates as a wrapper on top of `KeyValueStore` and uses only inside RadixHistory. If I will have the same parameters in `KeyValueStore` and `RadixStore` I don't see a reason to create `RadixStore`.